### ISSUE TITLE
Contact VA Screen - Accessibility - Text blocks Call VA311 and "VA311 is a national..." can not be read with screen reader in Android

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#17264]
## What happened?
I'm unable to get talkback to read this text on the Contact VA screen:
"CAL VA311

VA311 is a national toll-free number through which Veterans can access all VA has to offer."

When I tap or touch that part of the screen, the screen reader just gives me "Contact VA Screen."

## Specs:
- Device: Android phones Moto G7, Samsung Galaxy S7, Galaxy Note S9 (latest software)
- Talkback enabled

## Steps to Reproduce
- Enable talkback and navigate to the Contact VA screen.
- Attempt to read back the text noted above.

## Desired behavior
Everything on the screen may be read by the screen reader.

## Acceptance Criteria
- [ ] Everything on the screen may be read by the screen reader.

## Bug Severity
- 4 - Low

**Comment by esther-k** [comment]
**12/14**
Pass - Text blocks can be read by the screen reader on Android and iOS
